IoT/vss-extension.json

```diff
@@ -84,7 +83,7 @@
                   "ConnectedServiceName": "{{{assets.ARMEndpoint}}}",
                   "resourceGroupName": "{{{inputs.resourceGroup}}}",
                   "location": "{{{inputs.containerRegistryLocation}}}",
-                  "csmFileLink": "https://raw.githubusercontent.com/michaeljqzq/devops-iot-sample/7805ddfa886712eeb29fb3102de35f334a2ed95c/arm-acr.json",
+                  "csmFileLink": "https://raw.githubusercontent.com/Azure-Samples/devops-iot-scripts/arm/arm-acr.json",
```

Comment:
Two suggestions:
1. You could also put this file in source repository provided earlier. We have done that if you look at our pipeline templates. You can keep the ARM templates under a folder (e.g. armtemplates) side by side with application code under another folder (e.g. application). And in this input value you can specify the path to downloaded artifact directory.
2. If you still want to use direct url then commit-lock it. Otherwise any update if file can unintentionally have side effects

Reply:
fixed

Reply:
We prefer putting the script in one place to keep single source of truth. So host it somewhere else. Done with commit lock